### DISCUSSION TITLE
Help needed for installation under nobara linux

### DISCUSSION THREAD

**Discussion by jojo2203** [Cimbali/pympress#308] [Q&A]
Hello,
I would like to install pympress under nobra linux. When I enter :
```
dnf copr enable cimbali/pympress
dnf install python3-pympress
```

I only get the following error message: 
```
Unknown argument "copr" for command "dnf5". Add "--help" for more information about the arguments.
It could be a command provided by a plugin, try: dnf install dnf5-command(copr)
```

what am I doing wrong? thanks.

**Comment by Cimbali** [comment]
Those are the install instructions for fedora. I’m not familiar with nobara. If you can install the copr plugin (with the suggested `dnf install dnf5-command(copr)` then try that. [copr docs](https://docs.pagure.org/copr.copr/how_to_enable_repo.html) say the package is called `dnf-plugins-core`. If you can’t install the copr plugin and you want to add the fedora copr repo you can add the repo yourself using [the `.repo` file](https://copr.fedorainfracloud.org/coprs/cimbali/pympress/repo/fedora-rawhide/cimbali-pympress-fedora-rawhide.repo):

```
[copr:copr.fedorainfracloud.org:cimbali:pympress]
name=Copr repo for pympress owned by cimbali
baseurl=https://download.copr.fedorainfracloud.org/results/cimbali/pympress/fedora-$releasever-$basearch/
type=rpm-md
skip_if_unavailable=True
gpgcheck=1
gpgkey=https://download.copr.fedorainfracloud.org/results/cimbali/pympress/pubkey.gpg
repo_gpgcheck=0
enabled=1
enabled_metadata=1
```

It should be something like `dnf config-manager --add-repo /path/to/pympress.repo`

**Reply by jojo2203** [reply]
i installed dnf5-plugins with nobara packet manager. but it doesn't seem to find the python3-pympress package, see also output

https://copr.fedorainfracloud.org/api_3/rpmrepo/cimbali/pympress/nob 100% |   1.1 KiB/s |   1.0 KiB |  00m01sChroot not found in the given Copr project (nobara-39-x86_64).
You can choose one of the available chroots explicitly:
 centos-stream+epel-next-9-x86_64
 centos-stream-8-x86_64
 centos-stream-9-x86_64
 epel-8-x86_64
 epel-9-x86_64
 fedora-37-x86_64
 fedora-38-x86_64
 fedora-39-x86_64
 fedora-eln-x86_64
 fedora-rawhide-x86_64
 mageia-8-x86_64
 mageia-cauldron-x86_64
 openmandriva-cooker-x86_64
 openmandriva-rolling-x86_64
 opensuse-leap-15.3-x86_64
 opensuse-leap-15.4-x86_64
 opensuse-tumbleweed-x86_64
Updating and loading repositories:
 RPM Fusion for Fedora 39 - Nonfree - Updates                         100% | 102.4 KiB/s |   6.2 KiB |  00m00s
 Fedora 39 - x86_64 - Updates                                         100% |   4.1 KiB/s |   2.2 KiB |  00m01s
 nobara-baseos-multilib-39                                            100% |   2.1 KiB/s |   1.7 KiB |  00m01s
 RPM Fusion for Fedora 39 - Free - Updates                            100% | 120.6 KiB/s |   6.2 KiB |  00m00s
 nobara-appstream-39                                                  100% |   3.9 KiB/s |   3.2 KiB |  00m01s
 nobara-rocm-official                                                 100% |  10.1 KiB/s |   3.0 KiB |  00m00s
 nobara-baseos-39                                                     100% |   1.1 KiB/s |   1.7 KiB |  00m02s
Repositories loaded.
Failed to resolve the transaction:
No match for argument: python3-pympress

```

**Reply by Cimbali** [reply]
Did you manage to do the `enable cimbali/pympress` part? If that avenue fails, have you tried adding the `.repo` file manually?

**Comment by jojo2203** [comment]
No, unfortunately I was not successful, I was able to integrate the repo manually - but I am not a Linux professional either.

My solution was to switch to the Mandriva distro. The installation went smoothly with yay. I wanted to try out different distributions anyway (Debian, Ubuntu, Fedora, Nobara...)
